ansible/roles/osx-kvm/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+macos_version: 10.15.6
+macos_volume_size: 64G
+macos_disk_image: /opt/OSX-KVM/mac_hdd_ng.img
```

Comment:
Is there a reason to hardcode this path when we have a full-blown configuration management system (ansible) available to us now?

Reply:
No, there isn't, feel free to move to vars.